app/uk/gov/hmrc/plasticpackagingtax/registration/views/liability_process_more_weight_page.scala.html

```diff
@@ -0,0 +1,107 @@
+@*
```

Comment:
Not sure about the page name, does `liability_expect_to_exceed_threshold_in_30d` work better?